README.md

```diff
@@ -42,6 +42,8 @@ For Items, the fields are placed in the `properties`. For Collections, the field
 | sci:doi          | string               | The DOI of the data, e.g. `10.1000/xyz123`. This MUST NOT be a DOIs link. For all DOI names respective DOI links SHOULD be added to the links section (see chapter "Relation types"). |
 | sci:citation     | string               | The recommended human-readable reference (citation) to be used by publications citing the data. No specific citation style is suggested, but the citation should contain all information required to find the publication distinctively. |
 | sci:publications | [[Publication Object](#publication-object)] | List of relevant publications referencing and describing the data. |
+| sci:orcids        | \[string]            | An array of Open Researcher Contribution IDs ([ORCID](https://orcid.org)) associated with this product. For all ORCIDs a link SHOULD be added to the links section.     |
```

Comment:
Do orcid and ror need to be arrays? Isn't there usually one and then you may have one per asset in addition?